mock-certify-plugin/pom.xml

```diff
@@ -10,7 +10,7 @@
 
       <groupId>io.mosip.certify</groupId>
       <artifactId>mock-certify-plugin</artifactId>
-      <version>0.2.0</version>
+      <version>0.2.1-SNAPSHOT</version>
```

Comment:
This snapshot change should be done for all the projects in this repo, otherwise the build will fail. Since the github actions will try to  publish the dependencies to nexus snapshot repository. I believe we also need the github action change to hit the snapshot repo.

Please refer the below commit, yml file changes, we need the reverse of the same.
https://github.com/mosip/digital-credential-plugins/commit/232a6c6e8e38ce7a9601fba889602d2dda1b7839

Reply:
Changed to snapshot version for all projects and replaced release url with snapshot url.